README.md

```diff
@@ -2,138 +2,194 @@
-  },
-  transform: {
-    '^.+\\.stories\\.[jt]sx?$': '@storyboook/test-runner/jsdom/transform',
-    '^.+\\.[jt]sx?$': 'babel-jest',
-  },
-  testEnvironment: 'jest-environment-jsdom',
-};
+```jsx
+yarn storybook
 ```
 
-### Playwright
-
-Add a script to `package.json`:
+4. Run the test runner:
 
-```json
-{
-  "scripts": {
-    "test-storybook:playwright": "jest --config ./test-runner-jest.config.js"
-  }
-}
+```jsx
+yarn test-storybook --watch
 ```
 
-Then create a config file `test-runner-jest.config.js` that sets up your environment. For example:
+> **NOTE:** The runner assumes that your Storybook is running on `process.env.STORYBOOK_PORT` which defaults to `6006`. If you're running Storybook in another port, set the STORYBOOK_PORT before running your command like:
+>```jsx
+>STORYBOOK_PORT=9009 yarn test-storybook --watch
+>```
+
+## Configuration
+
+The test runner works out of the box, but you can override its configuration by adding a `test-runner-jest.config.js` that sets up your environment in the root folder of your project.
 
 ```js
+// test-runner-jest.config.js
 module.exports = {
-  cacheDirectory: '.cache/jest',
+  cacheDirectory: 'node_modules/.cache/storybook/test-runner',
   testMatch: ['**/*.stories.[jt]s(x)?'],
-  moduleNameMapper: {
-    // non-js files
-    '\\.(jpg|jpeg|png|apng|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
-      '@storybook/test-runner/mocks/fileMock.js',
-    '\\.(css|scss|stylesheet)$': '@storybook/test-runner/mocks/styleMock.js',
-  },
   transform: {
     '^.+\\.stories\\.[jt]sx?$': '@storybook/test-runner/playwright/transform',
     '^.+\\.[jt]sx?$': 'babel-jest',
   },
   preset: 'jest-playwright-preset',
   testEnvironment: '@storybook/test-runner/playwright/custom-environment.js',
+  testEnvironmentOptions: {
+    'jest-playwright': {
+      browsers: ['chromium', 'firefox', 'webkit'], // run tests against all browsers
+    },
+  },
 };
 ```
 
-> **NOTE:** The code currently assumes that your Storybook is ALREADY running on `process.env.STORYBOOK_PORT` which defaults to `6006`.
+The runner uses [jest-playwright](https://github.com/playwright-community/jest-playwright) and you can pass [testEnvironmentOptions](https://github.com/playwright-community/jest-playwright#configuration) to further configure it, such as how it's done above to run tests against all browsers instead of just chromium.
 
-#### Running against a deployed Storybook
+## Running against a deployed Storybook
 
-By default, the playwright assumes that you're running it against a locally served Storybook.
+By default, the test runner assumes that you're running it against a locally served Storybook.
 If you want to define a target url so it runs against deployed Storybooks, you can do so by passing the `TARGET_URL` environment variable:
 
+```bash
+TARGET_URL=https://the-storybook-url-here.com yarn test-storybook
+```
+
+## Running in CI
+
+### Running against deployed Storybooks on Github Actions deployment
```

Comment:
I thought of that but later I assumed people using chromatic would not need to do this because Chromatic essentially does it for them? So if I were using Chromatic and I wanted to save snapshots or something, I'd go for adding a CI step before chromatic that builds locally + tests and bails CI in case of failure, this way I make sure my stuff works before doing visual regression. WDYT?

Reply:
Makes sense. We can revisit later.